### ISSUE TITLE
Flake8 rule 'E901' reported an invalid point

### ISSUE THREAD

**Issue by ericjuma** [AtomLinter/linter-flake8#359]
Flake8 reported an invalid point for the rule `E901`, with the messge `IndentationError: expected an indented block`.

```py
from tempest.lib.services.volume.v3 import base_client
import tempest.test.BaseTestCase


class VolumesV3Test(tempest.test.BaseTestCase):

    def test_volume_create_get(self):
            
```

Debug information:
Atom version: 1.13.0
Flake8 version: `2.5.5 (pep8: 1.5.7, mccabe: 0.2.1, pyflakes: 0.8.1) CPython 2.7.13 on Darwin`

**Comment by Arcanemagus** [comment]
Please update your `flake8` version (`pip install -U flake8`). This particular error should be fixed in `flake8>=3.2.1`.

**Comment by Arcanemagus** [comment]
Closing due to a lack of response, if you are still having issues after updating `flake8` please provide code to reproduce this!
